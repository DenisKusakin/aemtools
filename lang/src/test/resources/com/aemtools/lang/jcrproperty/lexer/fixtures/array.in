[val1, val2, 3]
