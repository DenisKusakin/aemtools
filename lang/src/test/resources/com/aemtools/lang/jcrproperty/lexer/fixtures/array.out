[Jp] [ ('[')
[Jp] ARRAY_VALUE_TOKEN ('val1')
[Jp] , (',')
WHITE_SPACE (' ')
[Jp] ARRAY_VALUE_TOKEN ('val-2')
[Jp] , (',')
WHITE_SPACE (' ')
[Jp] ARRAY_VALUE_TOKEN ('3')
[Jp] , (',')
WHITE_SPACE (' ')
[Jp] ARRAY_VALUE_TOKEN ('jcr:property')
[Jp] ] (']')
